src/main/java/spireMapOverhaul/zones/volatileGrounds/monsters/GremlinArchmage.java

```diff
@@ -0,0 +1,183 @@
+package spireMapOverhaul.zones.volatileGrounds.monsters;
+
+import basemod.abstracts.CustomMonster;
+import com.badlogic.gdx.math.MathUtils;
+import com.esotericsoftware.spine.AnimationState;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.AnimateFastAttackAction;
+import com.megacrit.cardcrawl.actions.animations.AnimateJumpAction;
+import com.megacrit.cardcrawl.actions.common.*;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.cards.status.Burn;
+import com.megacrit.cardcrawl.cards.status.Slimed;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.MonsterStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.monsters.beyond.Exploder;
+import com.megacrit.cardcrawl.monsters.beyond.SnakeDagger;
+import com.megacrit.cardcrawl.powers.*;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.zones.volatileGrounds.powers.ExplodePlusPower;
+
+import java.util.Iterator;
+
+public class GremlinArchmage extends CustomMonster {
+    public static final String ID = SpireAnniversary6Mod.makeID("GremlinArchmage");
+    private static final MonsterStrings monsterStrings = CardCrawlGame.languagePack.getMonsterStrings(ID);
+    public static final String NAME = monsterStrings.NAME;
+    public static final String[] MOVES = monsterStrings.MOVES;
+    private static final String TEMP_IMG = SpireAnniversary6Mod.makeImagePath("monsters/VolatileGrounds/GremlinArchmage/GremlinArchmage.png");
+    //TODO: ADD ANIMATION
```

Comment:
I assume you're planning to implement this in prior to release. Let me know if there's a point where you decide to stop working on the remaining TODOs and just want to get this ready to be merged.

Reply:
Anything I push is ready to be released (after reviews of course), the TODO parts are non-essential.
I wish there was an alternate "Implement later" annotation for that sort of stuff tbh.